commands/server/init.go

```diff
@@ -92,7 +103,16 @@ func addGenesisOptions(filename string, options json.RawMessage) error {
 		return err
 	}
 
+	v, ok := doc[AppStateKey]
```

Comment:
good idea with the force flag, and not just prevent the app_state override always